packages/hub/test/scope.test.ts

```diff
@@ -129,6 +129,12 @@ describe('Scope', () => {
       expect((scope as any)._span).toEqual(undefined);
     });
 
+    test('setProcessingMetadata', () => {
+      const scope = new Scope();
+      scope.setProcessingMetadata({ dogs: 'are great!' });
+      expect((scope as any)._processingMetadata.dogs).toEqual('are great!');
```

Comment:
Except that the method doesn't overwrite the entire value, it just sets a key-value pair. In this case, since it starts out empty, they're the same thing, of course, but IMHO the existing version is truer to the how the function actually behaves.